DataCapturing/DistanceCalculationStrategy.swift

```diff
@@ -25,27 +25,36 @@ import CoreLocation
  Different implementations are possible.
 
  - Author: Klemens Muthmann
- - Version: 1.0.0
+ - Version: 1.1.0
  - Since: 2.2.0
  */
 public protocol DistanceCalculationStrategy {
     /**
      Calculate the distance in meters between the provided `CLLocation` instances.
 
      - Parameters:
-     - from: The location to start the distance calculation from
-     - to: The location to calculate the distance to
+        - from: The location to start the distance calculation from
+        - to: The location to calculate the distance to
      */
     func calculateDistance(from previousLocation: CLLocation, to location: CLLocation) -> Double
 
     /**
      This is similar to `calculateDistance(:CLLocation,:CLLocation)` but accepts `GeoLocationMO` arguments.
 
      - Parameters:
-     - from: The location to start the distance calculation from
-     - to: The location to calculate the distance to
+        - from: The location to start the distance calculation from
+        - to: The location to calculate the distance to
      */
     func calculateDistance(from previousLocation: GeoLocationMO, to location: GeoLocationMO) -> Double
+
+    /**
+     Calculate the distance between two coordinate pairs
+
+     - Parameters:
+        - from: The source coordinate pair
+        - to: The target coordinate pair
+     */
+    func calculateDistance(from previousLocationLatLonCoordinates: (Double, Double), to locationLatLonCoordinates: (Double, Double)) -> Double
```

Comment:
This might not be familiar to Java developers. The notation (Double, Double) is a tuple of two double numbers. (A poor mans value object) ;)